lib/Elastica/Index.php

```diff
@@ -531,10 +524,19 @@ public function request($path, $method, $data = array(), array $query = array())
      *
      * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-analyze.html
      */
-    public function analyze($text, $args = array())
+    public function analyze($text, $args = [])
     {
         $data = $this->request('_analyze', Request::POST, $text, $args)->getData();
 
         return $data['tokens'];
     }
+
+    private function silenceDelete()
```

Comment:
Is there any use in exposing this functionality as part of a parameter to the delete() method instead?